eslint.config.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const neo = require('neostandard')
+
+module.exports = [
+  ...neo({
+    env: ['jest'],
+    ignores: [
+      'lib/llhttp',
+      'test/fixtures/wpt',
+      'node_modules',
```

Comment:
Not needed, this is already part of ESLint's default ignore: https://github.com/eslint/eslint/blob/21d3766c3f4efd981d3cc294c2c82c8014815e6e/lib/config/default-config.js#L59